brayns/common/camera/Camera.cpp

```diff
@@ -51,10 +51,13 @@ struct Camera::Impl
         const Vector3f& target,
         const Vector3f& upVector )
     {
+        const Vector3f& p = getPosition();
+        const Vector3f& t = getTarget();
+        const Vector3f& u = getUpVector();
         setPosition( position );
         setTarget( target );
         setUpVector( upVector );
-        modified = true;
+        modified = ( p != position || t != target || u != upVector );
```

Comment:
yes, then the problem should be addressed in each of the individual setter, they must do (example code):
```
setPosition( newPos ){
if( newPos == _pos )
    return();
_pos = newPos;
}
```